MLendPoint/src/app.js

```diff
@@ -9,7 +10,8 @@ app.use(cors());
 
 const port = process.env.PORT || 8080;
 
-app.use(express.static("../public/"));
+app.use(express.static(path.join(__dirname, '../public')));
```

Comment:
What's exactly doing this method? Could you confirm if this create in other path the files uploads?

Reply:
This method is more specific to get public path, path.join help to concatenate __dirname + '..public'